pyvisa/util.py

```diff
@@ -1057,3 +1057,32 @@ def get_arch(filename: Union[str, Path]) -> List[ArchitectureType]:
             archs.append(ArchitectureType.AARCH64)
 
         return archs
+
+
+def message_size(
+    num_points: int,
+    point_size: Literal[1, 2] = 2,
```

Comment:
Here people may need to use 4 or 8 when transferring floating points number. I wonder if proving the format of the data expected (b, h, f, d, etc) and doing the conversion internally would not be nicer from a user perspective.

Reply:
Done.  I changed the parameter from "point_size: int" to "datatype: BINARY_DATATYPES" and used the struct module to determine the point size from the BINARY_DATATYPES code.